snowpack/src/util.ts

```diff
@@ -313,18 +313,27 @@ export function isPackageAliasEntry(val: string): boolean {
 
 /** Get full extensions of files */
 export function getExt(fileName: string) {
-  return {
-    /** base extension (e.g. `.js`) */
-    baseExt: path.extname(fileName).toLocaleLowerCase(),
-    /** full extension, if applicable (e.g. `.proxy.js`) */
-    expandedExt: path.basename(fileName).replace(/[^.]+/, '').toLocaleLowerCase(),
-  };
+  const exts: string[] = [];
+  let current = '', mat;
+  while (mat = /\.[^.\\\/]*$/.exec(fileName)) {
+    current = mat[0] + current;
+    fileName = fileName.substr(0, mat.index);
+    exts.unshift(current);
+  }
+  return exts;
+}
+
+export function getLastExt(fileName: string) {
+  const mat = /\.[^.\\\/]*$/.exec(fileName);
```

Comment:
What’s the difference between this and `path.extname()`?

Reply:
I ask because ideally this behavior is combined with `getExt()`. FWIW we aren’t using `getExt().expandedExt` anywhere, so you should be able to combine these utilities together. When other contributors work with this it’s just one less function to have to deal with (even internally we’ve had confusion over “when do I use X vs Y?”).

Reply:
@drwpow Thanks! I guess there's no difference, really. This can be replaced with `extname()`.

But there's no `getExt().expandedExt`:
```
getExt("path/to/file.partial.temp.example.html.js") === [
  ".partial.temp.example.html.js",
  ".temp.example.html.js",
  ".example.html.js",
  ".html.js",
  ".js",
]
```

Reply:
@drwpow I tried just now replacing `getLastExt()` with `path.extname()`, but it caused many errors.

Here's one example:

```
getLastExt(".js") === ".js"; // OK
path.extname(".js") === ""; // Causes errors
```